### PR TITLE
feature/Rewrite-PlayRoom-to-JSX.

### PR DESCRIPTION
**Pull request by nddmanh**

### CODE REVIEW

File: src/modules/playRoom/components/BetMoneyModal.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    background: #141414;
+    color: #fff;
+    border: none;
+    border-radius: 10px;
+    margin-top: 20px;
+  }
+`;
+
+const ModalInput = styled.input`
+  line-height: 1.8;
```
Comment (ppppbn) [inline src/modules/playRoom/components/BetMoneyModal.js]: Why line-height has the value of 1.8?

**Comment by nddmanh** [comment]
UI Play Room:
image


When player click a pick:
image

File: src/modules/playRoom/components/PickList.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickList.js]: remove React here

File: src/modules/playRoom/components/PickList.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      id: 5,
+      src: "./image-pick/nai.png",
+      value: "Nai",
+    },
+    
+    { 
+      id: 6,
+      src: "./image-pick/bau.png",
+      value: "Bau",
+    },
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickList.js:43]: Please put the default value of image list in a constant

File: src/modules/playRoom/components/PickList.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  ]);
+
+  return (
+    <>
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickList.js]: no need fragment div here

File: src/modules/playRoom/components/PickList.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      <div id="game">
+        <Row justify="center" gutter={[0, 24]} >
+          {
+            imagesPick.map((img) => (
+              <Col xs={24} sm={12} md={12} xl={8} lg={8} >
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickList.js]: lg={8} is enough. remove xl={8}

File: src/modules/playRoom/pages/PlayRoom.css [hunk 6]
```diff
@@ -0,0 +1,34 @@
+
+#game {
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.css]: we don't use id for css

File: src/modules/playRoom/components/PickList.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+      value: "Bau",
+    },
+  ]);
+
+  return (
+    <>
+      <div id="game">
+        <Row justify="center" gutter={[0, 24]} >
+          {
+            imagesPick.map((img) => (
```
Comment (ppppbn) [inline src/modules/playRoom/components/PickList.js]:
```suggestion
            imagesPick.map(img => (
```

File: src/modules/playRoom/pages/PlayRoom.css [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    flex-direction: row;
+    border-radius: 10px;
+    padding: 55px 20px 50px 20px;
+    margin-top: 30px;
+}
+
+.select {
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.css]: this class name can be mistakenly overridden. you should change the name

File: src/modules/playRoom/pages/PlayRoom.js [hunk 9]
```diff
@@ -1,8 +1,21 @@
...
+import PickList from '../components/PickList';
+import './PlayRoom.css';
 
-function PlayRoom(props) {
+function PlayRoom() {
   return (
-    <div></div>
+    <Row justify="center" gutter={[32, 32]} >
+      <Col xs={24} sm={24} md={24} xl={6} > 
+
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.js]: Why do we have two Col with no content inside?

File: src/modules/playRoom/pages/PlayRoom.js [hunk 10]
```diff
@@ -1,8 +1,21 @@
...
+import { Row, Col} from 'antd';
+import PickList from '../components/PickList';
+import './PlayRoom.css';
 
-function PlayRoom(props) {
+function PlayRoom() {
   return (
-    <div></div>
+    <Row justify="center" gutter={[32, 32]} >
+      <Col xs={24} sm={24} md={24} xl={6} >
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.js]:
```suggestion
      <Col xs={24} xl={6} > 
```

File: src/modules/playRoom/pages/PlayRoom.js [hunk 11]
```diff
@@ -1,8 +1,21 @@
...
 
-function PlayRoom(props) {
+function PlayRoom() {
   return (
-    <div></div>
+    <Row justify="center" gutter={[32, 32]} >
+      <Col xs={24} sm={24} md={24} xl={6} > 
+        
+      </Col>
+      <Col xs={24} sm={24} md={24} xl={12} >
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.js]:
```suggestion
      <Col xs={24} xl={12} > 
```

**Comment by ppppbn** [comment]
The last commit has non meaningful message. Please remove or change the commit message

**Comment by ppppbn** [comment]
Also this PR has too many commits. Try group them into one as they all serve one purpose: UI for PlayRoom

File: src/modules/playRoom/components/PickItem.js [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >                
+        <input type="image" className="img-select" src={img.src} alt="" name="" value={img.value} />
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js]: Do not leave attributes empty, remove it if not necessary

File: src/modules/playRoom/pages/PlayRoom.js [hunk 13]
```diff
@@ -1,8 +1,21 @@
 import React, { useState, useEffect } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.js]: Remove React, useState, useEffect

File: src/modules/playRoom/pages/PlayRoom.css [hunk 14]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.css]: Remember we use ```scss``` in this project not ```css```

File: src/modules/playRoom/components/PickItem.js [hunk 15]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js]: Remove React

**Pull request closed without merging**